recipes/dylib/all/conanfile.py

```diff
@@ -0,0 +1,50 @@
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import copy, get
+from conan.tools.layout import basic_layout
+import os
+
+required_conan_version = ">=1.53.0"
+
+class DylibConan(ConanFile):
+    name = "dylib"
+    homepage = "https://github.com/martin-olivier/dylib"
+    url = "https://github.com/conan-io/conan-center-index"
+    author = "Martin Olivier (martin.olivier@live.fr)"
```

Comment:
```suggestion
```

The author attribute is not allowed, because recipe is owned by the Conan community :)